.github/workflows/pytest-core-nompi.yml

```diff
@@ -36,102 +35,104 @@ jobs:
            py36-gcc7-openmp,
            py37-gcc7-noopenmp,
            py37-gcc8-openmp,
-           py38-gcc9-openmp
+           py38-gcc9-openmp,
+           py37-osx,
+           docker-devito
         ]
         include:
-            - name: py36-gcc49-openmp
-              python-version: 3.6
-              gcc-version: 4.9
-              openmp: 1
-
-            - name: py37-gcc5-openmp
-              python-version: 3.7
-              gcc-version: 5
-              openmp: 1
-
-            - name: py38-gcc6-openmp
-              python-version: 3.8
-              gcc-version: 6
-              openmp: 1
-
-            - name: py36-gcc7-openmp
-              python-version: 3.6
-              gcc-version: 7
-              openmp: 1
-
-            - name: py37-gcc7-noopenmp
-              python-version: 3.7
-              gcc-version: 7
-              openmp: 0
-
-            - name: py37-gcc8-openmp
-              python-version: 3.7
-              gcc-version: 8
-              openmp: 1
-
-            - name: py38-gcc9-openmp
-              python-version: 3.8
-              gcc-version: 9
-              openmp: 1
+        - name: py36-gcc49-openmp
+          python-version: 3.6
+          os: ubuntu-16.04
+          arch: "gcc-4.9"
+          openmp: 1
+
+        - name: py37-gcc5-openmp
+          python-version: 3.7
+          os: ubuntu-16.04
+          arch: "gcc-5"
+          openmp: 1
+
+        - name: py38-gcc6-openmp
+          python-version: 3.8
+          os: ubuntu-16.04
+          arch: "gcc-6"
+          openmp: 1
+
+        - name: py36-gcc7-openmp
+          python-version: 3.6
+          os: ubuntu-16.04
+          arch: "gcc-7"
+          openmp: 1
+
+        - name: py37-gcc7-noopenmp
+          python-version: 3.7
+          os: ubuntu-16.04
+          arch: "gcc-7"
+          openmp: 0
+
+        - name: py37-gcc8-openmp
+          python-version: 3.7
+          os: ubuntu-16.04
+          arch: "gcc-8"
+          openmp: 1
+
+        - name: py38-gcc9-openmp
+          python-version: 3.8
+          os: ubuntu-16.04
+          arch: "gcc-9"
+          openmp: 1
+
+        - name: py37-osx
```

Comment:
it's a bit weird that we call this build `-osx` , so we suffix it with the OS name, while the other ones with the backend compiler `gcc8` `gcc9` etc

perhaps the others should be `py37-linux-gcc8-omp` (note: also renaming openmp as omp)

Reply:
Can make a renaming pass

Reply:
renamed